install/charts/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Karydia
+name: karydia
+version: 0.1.0
```

Comment:
Maybe we should align this version with the VERSION file and automatic CI commits. What's the semantic difference between `version` and `appVersion`?

Reply:
"version" seems to be the version of the helm chart itself.
"appVersion" should be the version of the application that is deployed by the helm chart.
Read https://github.com/helm/helm/blob/master/docs/charts.md for the documentation.

Reply:
For now, it should be sufficient to just remove the appVersion as it is already saved in the VERSION file.